js_modules/dagster-ui/packages/ui-components/src/components/useDelayedState.tsx

```diff
@@ -1,12 +1,12 @@
 import {useEffect, useState} from 'react';
 
 export const useDelayedState = (delayMsec: number) => {
-  const [value, setValue] = useState(false);
+  const [ready, setReady] = useState(false);
 
   useEffect(() => {
-    const timer = setTimeout(() => setValue(true), delayMsec);
+    const timer = setTimeout(() => setReady(true), delayMsec);
     return () => clearTimeout(timer);
   }, [delayMsec]);
 
-  return value;
+  return ready;
```

Comment:
Renamed this var because I found that it's not clear what "value" is supposed to mean.